.github/workflows/email.yml

```diff
@@ -3,21 +3,25 @@ name: email
 on:
   push:
     branches:
-      - main 
+     # - main 
```

Comment:
Commented out to see if this works when we merge a PR to main from forked repository.
If this patch works will remove this commented line.